### ISSUE TITLE
demoHMFdays.m

### ISSUE THREAD

**Issue by jvoight** [edgarcosta/hilbertmodularforms#232]
There's something in the top level directory which doesn't run any more (issues with characters).  

It seems to be newer than the one in the example files with the same name.  

I don't think we should clutter the top directory, we can just say in the readme file what to paste into magma if you're a noob.

**Comment by SamSchiavone** [comment]
Right, there's also a file called usage.m in the examples directory that also doesn't work anymore. I think at the moment the best way to see how the code works is to look at files in the Tests directory.
